srcs/client/src/Pages/Profile/Fetch/requestFriendship.tsx

```diff
@@ -0,0 +1,24 @@
+
+export async function requestFriendship(props : any){
+	try{
+		await
+		fetch(props.url, {
+	  method: "PUT",
+	  credentials: "include",
+	  headers: {
+		Accept: "application/json",
+		"Content-Type": "application/json",
+		"Access-Control-Allow-Credentials": "true",
+	  },
+	  body: JSON.stringify({ addresseeId: props.addresseeId })
+	}).then((response) =>{
+			if (response.status !== 200){
+				console.error(response.status);
+				throw new Error("Error");
```

Comment:
I think you can use response.message if you want a more specific error message. We are providing one from the backend.